CrewChiefV4/Events/Fuel.cs

```diff
@@ -530,6 +573,22 @@ public override void respond(String voiceMessage)
             {
                 reportFuelStatus();
             }
+            else if (SpeechRecogniser.ResultContains(voiceMessage, SpeechRecogniser.CALCULATE_FUEL_FOR))
+            {
+                int units = 0;
+                foreach (KeyValuePair<String, int> entry in SpeechRecogniser.numberToNumber)
+                {
+                    if (voiceMessage.Contains(" " + entry.Key))
+                    {
+                        units = entry.Value;
+                        break;
+                    }
+                }
+                if (!reportFuelConsumptionForLaps(units))
```

Comment:
need to check whether we're asking for laps or time (hours / minutes) here?

Reply:
Yes It will be added if message.contains(LAPS || LAP) else....